checkoutmanager/dirinfo.py

```diff
@@ -156,6 +185,24 @@ class BzrDirInfo(DirInfo):
     vcs = 'bzr'
 
     @capture_stdout
+    def cmd_rev(self):
+        print(self.directory)
+        os.chdir(self.directory)
+        print(system("bzr log -r-1"))
+
+    @capture_stdout
+    def cmd_cu(self):
+        output = system("bzr missing")
+        output = output.strip()
+        output_lines = output.split('\n')
+        if "Branches are up to date." in output_lines:
```

Comment:
I agree. I've tried to use the most apparently reliable indicators for each command, but my experience with both bzr and hg was some years ago. 

Since these commands are alongside the checkoutmanager functionality that exists at present and don't interfere with it otherwise, I propose starting out with something which seems to be acceptable and tweak it as and when issues are discovered. 

If there is a more reliable indicator, I'm more than happy to switch to that. For example, I'm currently considering ``bzr revno`` for ``up`` instead of ``bzr log -r-1``. This should more reliably return the revision number.